docs/hooks.md

```diff
@@ -0,0 +1,81 @@
+# Introduction
+
+Host OS update hooks run on devices to configure the device before a Host OS Update. They usually handle the boot partition or any migrations between various OS versions.
+
+resinHUP runs the `hostapp-update` script on device with the right flags that point to the new OS docker tag.
+
+# Hooks v1.0 flow
+- __v2.43 and before support only Hooks v1.0 flow__
```

Comment:
I'd also avoid having explicit versions here especially that you don't know what versionist will set for the `next` release.

Reply:
Removed mentions